presto-hive/src/main/java/com/facebook/presto/hive/HiveWriteUtils.java

```diff
@@ -229,6 +236,9 @@ public static Object getField(Type type, Block block, int position)
         if (type instanceof VarcharType) {
             return new Text(type.getSlice(block, position).getBytes());
         }
+        if (type instanceof CharType) {
+            return new Text(type.getSlice(block, position).getBytes());
```

Comment:
Do we need to materialize the trailing spaces here? I notice that the read path strips out the spaces. Are they suppose to be there or not for Hive?